src/OrchardCore.Modules/OrchardCore.Workflows/Views/Items/WorkflowFaultEvent.Fields.Edit.cshtml

```diff
@@ -0,0 +1,57 @@
+@using OrchardCore.Workflows.Models
+@model WorkflowFaultViewModel
+ 
+ <div class="form-group" asp-validation-class-for="ErrorFilter">
+    <label asp-for="ErrorFilter">@T["Trigger Condition"]</label>
+    <div id="@Html.IdFor(x => x.ErrorFilter)_editor" asp-for="Text" style="min-height: 400px;" class="form-control"></div>
+     <textarea asp-for="ErrorFilter" hidden>@Html.Raw(Model.ErrorFilter)</textarea>
+    <span asp-validation-for="ErrorFilter"></span>
+    <span class="hint">@T["A collection of workflow names that need to return a collection of strings. Javascript syntax."]</span><br />
+    <span class="hint">@T["Get error information： input('{0}')", WorkflowFaultModel.WorkflowFaultInputKey]</span><br />
+    <span class="hint">@T["Contains the following attributes："]</span><br />
+    <span class="hint">@T["{0}：workflow Id", nameof(WorkflowFaultModel.WorkflowId)]</span><br />
+    <span class="hint">@T["{0}：Workflow name", nameof(WorkflowFaultModel.WorkflowName)]</span><br />
+    <span class="hint">@T["{0}：Number of executed nodes", nameof(WorkflowFaultModel.ExcutedActivityCount)]</span><br />
+    <span class="hint">@T["{0}：The name of the node", nameof(WorkflowFaultModel.ActivityDisplayName)]</span><br />
+    <span class="hint">@T["{0}：Node Type Name", nameof(WorkflowFaultModel.ActivityTypeName)]</span><br />
+    <span class="hint">@T["{0}：Node Id", nameof(WorkflowFaultModel.ActivityId)]</span><br />
+    <span class="hint">@T["{0}：Exception Information Summary", nameof(WorkflowFaultModel.ErrorMessage)]</span><br />
+    <span class="hint">@T["{0}：Exception Details", nameof(WorkflowFaultModel.ExceptionDetails)]</span><br />
+    <span class="hint">@T["{0}：Contains error messages manually raised by workflow nodes", nameof(WorkflowFaultModel.FaultMessage)]</span><br />
+ </div>
+
+<script asp-name="monaco" depends-on="admin" at="Foot"></script>
+<script at="Foot" depends-on="monaco">
+    $(document).ready(function () {
+        require(['vs/editor/editor.main'], function () {
+            var settings=  {
+                  "automaticLayout": true,
+                  "language": "javascript"
+                };
+
+            var html = document.getElementsByTagName("html")[0];
+            const mutationObserver = new MutationObserver(setTheme);
+            mutationObserver.observe(html, { attributes: true });
+
+            function setTheme() {
+                var theme = html.dataset.theme;
+                if (theme === "darkmode") {
+                    monaco.editor.setTheme('vs-dark')
```

Comment:
Why you set the theme here, @Skrypt please your input here while your worked on the themes

Reply:
For darkmode switch

if remove that 
![image](https://user-images.githubusercontent.com/15613121/184533855-f6149c59-27db-47e4-84f4-145694772f24.png)


Refer to the usage of other positions, which should be

![image](https://user-images.githubusercontent.com/15613121/184533899-3d7f9a33-9489-4f9c-9655-af2ea85a3c39.png)